plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/GeneralUtils.java

```diff
@@ -527,6 +530,16 @@ public static String replaceVariables(@NotNull String string, IVariableResolver
         }
     }
 
+    @NotNull
+    private static String substituteVariable(@NotNull String string, @NotNull Matcher matcher, @NotNull String varValue) {
+        if (matcher.start() == 0 && matcher.end() >= string.length() - 1) {
+            string = varValue;
+        } else {
+            string = string.substring(0, matcher.start()) + varValue + string.substring(matcher.end());
+        }
+        return string;
```

Comment:
```suggestion
        if (matcher.start() == 0 && matcher.end() >= string.length() - 1) {
            return varValue;
        } else {
            return string.substring(0, matcher.start()) + varValue + string.substring(matcher.end());
        }
```

Reply:
I don't think that multiple exit points is a good practice in a common case. There is no reason to do it in this method.

Reply:
I don't think that multiple exit points is a good practice in a common case. There is no reason to do it in this method.

Reply:
Reassigning an argument is neither a good practice.

Reply:
OK